tests/PHPUnit/Integration/ArchiveProcessor/LoaderTest.php

```diff
@@ -743,24 +745,6 @@ public function getTestDataForArchiving()
                         'name' => 'nb_visits',
                         'value' => '1',
                     ),
-                    array (
```

Comment:
Those archives are now correctly removed even if they are created in the same second.